docs/getting-started/install-configure.md

```diff
@@ -5,16 +5,39 @@ weight: 2
 indent: true
 ---
 
-# Install & Configure Crossplane
+# Choosing Hosted vs. Self-Hosted Crossplane
 
-Crossplane can be easily installed into any existing Kubernetes cluster using
-the regularly published Helm chart. The Helm chart contains all the custom
-resources and controllers needed to deploy and configure Crossplane.
+Users looking to use Crossplane for the first time have two options avaiable to them 
+today. The easiest way to start using Crossplane quickly is to use a hosted Crossplane 
+service like Upbound Cloud. Setup should take 10-15 minutes total.
```

Comment:
> Setup should take 10-15 minutes total.

How would you feel about removing this part? Partly because it crosses the line into feeling a bit like it's selling Upbound Cloud, and partly because 10-15 mins is probably about the same as (or longer than) the time most folks with an existing Kubernetes cluster would need to get setup with Crossplane.

Reply:
That's very fair and good feedback. Would something like this be more appropriate instead?

> Users looking to use Crossplane for the first time have two options avaiable to
> them today. The first way is to use a hosted Crossplane service like Upbound
> Cloud. Alternatively, users looking for some more flexability can install
> Crossplane into their own Kubernetes cluster.
> 
> Crossplane will be installed using the regularly published Helm chart. The Helm
> chart contains all the custom resources and controllers needed to deploy and
> configure Crossplane.
>